sp21/group.yml

```diff
@@ -1,11 +1,13 @@
 # Check the Docs for details on editing this!
 name: "discussions"
-room: ""
+room: "Online"
 semester: "sp21"
-startdate: "2021-01-18T00:00:00"
-directors: []
-coordinators: []
-guests: []
+startdate: "2021-01-19T18:00:00"
```

Comment:
Discussions starts next week, right?

Reply:
Yeah sorry i had set the first interest meeting as the first meeting changing it now